backend/tracim_backend/lib/utils/utils.py

```diff
@@ -392,6 +393,31 @@ def email_link(self) -> str:
         return "mailto:{email_address}".format(email_address=self.email_address)
 
 
+def load_apps() -> typing.List[TracimApplication]:
```

Comment:
I'm not sure load_apps function should be in `tracim_backend.lib/utils.utils`... this is a critical core feature

Reply:
don't really know where we should but this, as tracim_backend.lib.core is for lib like ContentApi, etc...
we could also put if tracim_backend.lib.<something_else> or at rhigher level tracim_backend.<something> like config, error, hookspec.

Reply:
This can be a class function of `ApplicationApi`

Reply:
Or this function can stay a util function, but take module to "explore" as parameter.

Reply:
moved to tracim_backend.apps